service/src/java/org/apache/hive/service/cli/thrift/ThriftHttpServlet.java

```diff
@@ -143,6 +140,7 @@ public ThriftHttpServlet(TProcessor processor, TProtocolFactory protocolFactory,
     if (this.authType.isEnabled(HiveAuthConstants.AuthTypes.JWT)) {
       this.jwtValidator = new JWTValidator(hiveConf);
     }
+    this.httpAuthService = new HttpAuthService(cookieDomain, cookiePath, cookieMaxAge, isCookieSecure, AUTH_COOKIE);
```

Comment:
I can see a few cookie related parameters here passed to auth service
after refactoring this whole auth thing to a separate auth service, does it make sense to keep those in ThriftHttpServlet at all as fields?

Reply:
Done

Reply:
ack